lib/gmail/gmail.go

```diff
@@ -106,7 +131,15 @@ func NewGmail(dir string, label string) (*Gmail, error) {
 	} else {
 		g.cache = gmailCache{c}
 	}
-	clt, err := newClient(&g)
+	var clt *http.Client
+	var err error
+	if len(serviceAccountJSONFile) != 0 && len(toImpersonate) != 0 {
```

Comment:
I don't understand this conditional (`&& len(toImpersonate) !=0`).

The `newJWTClient` definition seems to imply it works fine without specifying an account to impersonate. Shouldn't this only check if `len(serviceAccountJSONFile)` is nonempty?